lighthouse-core/test/report/v2/report-generator-test.js

```diff
@@ -143,6 +146,20 @@ describe('ReportGeneratorV2', () => {
       assert.ok(result.includes('LIGHTHOUSE_JAVASCRIPT'), 'cannot be tricked');
     });
 
+    it('should inject the report templates', () => {
+      const page = jsdom.jsdom(new ReportGeneratorV2().generateReportHtml({}));
+      const templates = jsdom.jsdom(TEMPLATES_FILE);
+
+      assert.deepEqual(page.querySelectorAll('template[id^="tmpl-"]'),
```

Comment:
`deepStrictEqual`? Deep equality checking also gets weird with e.g. empty objects and properties on prototypes ([it doesn't check them](https://nodejs.org/api/assert.html#assert_assert_deepequal_actual_expected_message)), so it may be worth doing an explicit comparison with the expected elements